content/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -0,0 +1,222 @@
+and test your credentials to ensure your applications, agents, and visualization tools can connect to InfluxDB.
+
+If you upgrade with `auth-enabled = false`, the upgrade may appear complete,
+but client requests to InfluxDB 2.0 may be silently ignored (you won't see a notification the request was denied).
+
+### In-memory indexing option
+
+InfluxDB 2.0 doesn't support [in-memory (`inmem`) indexing](/influxdb/v1.8/administration/config/#in-memory-inmem-index-settings). The following InfluxDB 1.x configuration options associated with `inmem` indexing are ignored in the upgrade process:
+
+- `max-series-per-database`
+- `max-values-per-tag`
+
+## Perform the upgrade
+
+If you've considered the [guidance above](#before-you-begin-important-considerations)
+and are ready to proceed, follow these steps to upgrade your InfluxDB 1.x to InfluxDB 2.0.
+
+1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/).
+   Unpackage the InfluxDB binaries and place them in your `$PATH`.
+2. Stop your running InfluxDB 1.x instance.
+   Make a backup copy of all 1.x data before upgrading:
+
+   ```sh
+   cp -R .influxdb/ .influxdb_bak/
+   ```
+3. Use `influxd version` to ensure you are running InfluxDB 2.0 from the command line.
+   The `influxd upgrade` command is only available in InfluxDB 2.0.
+4. If your 1.x configuration file is at the [default location](/influxdb/v1.8/administration/config/#using-the-configuration-file), run:
+
+    ```sh
+    influxd upgrade
+    ```
+
+    If your 1.x configuration file is not at the default location, run:
+
+    ```sh
+    influxd upgrade --config-file <path to v1 config file>
+    ```
+
+    To store the upgraded 2.0 configuration file in a custom location, include the `--v2-config-path` flag:
+
+    ```sh
+    influxd upgrade --v2-config-path <destination path for v2 config file>
+    ```
+
+5. Follow the prompts to set up a new InfluxDB 2.0 instance.
+
+   ```
+   Welcome to InfluxDB 2.0 upgrade!
+   Please type your primary username: <your-username>
+
+   Please type your password:
+
+   Please type your password again:
+
+   Please type your primary organization name: <your-org>
+
+   Please type your primary bucket name: <your-bucket>
+
+   Please type your retention period in hours.
+   Or press ENTER for infinite:
+
+   You have entered:
+     Username:          <your-username>
+     Organization:      <your-org>
+     Bucket:            <your-bucket>
+     Retention Period:  infinite
+   Confirm? (y/n): y
+   ```
+
+The output of the upgrade prints to standard output.
+It is also saved (for troubleshooting and debugging) in the current directory to a file called `upgrade.log` located in the home directory of the user running `influxdb upgrade`.
+
+### Post-upgrade
+
+To verify 1.x users were successfully migrated to 2.0, run [`influx v1 auth list`](/influxdb/v2.0/reference/cli/influx/v1/auth/list/).
+
```

Comment:
I think this could be misleading or confusing. You can technically use a token with the compatibility API (arbitrary username, token as the password).